src/Traits/ConditionTrait.php

```diff
@@ -24,10 +24,6 @@ trait ConditionTrait
     #[ORM\Column(type: 'json', nullable: true)]
     protected ?array $value = null;
 
-    #[ORM\ManyToOne(targetEntity: ValidatorInterface::class, cascade: ['persist'], inversedBy: 'conditions')]
-    #[ORM\JoinColumn(onDelete: 'CASCADE')]
-    protected ?ValidatorInterface $validator = null;
-
```

Comment:
Looks good. Just need to add this to the Upgrade.md

Reply:
Do you mean in case someone implemented a custom validator and needs to add the property now? Or what else would need to be covered in the Upgrade.md file?

I assume you mean the `UPGRADE-4.md` file, right?

Reply:
Yep exactly and the file name is fine.

Reply:
Will take care of this in the next few days. Thanks for your insights!